pkg/server/service.go

```diff
@@ -233,7 +253,7 @@ func (c *criService) Close() error {
 	if err := c.streamServer.Stop(); err != nil {
 		return errors.Wrap(err, "failed to stop stream server")
 	}
-	return nil
+	return c.cniNetConfMonitor.stop()
```

Comment:
let's wrap the error with something like  "failed to stop cniNetConfMonitor"

Reply:
it feels weird to possibly be doing a cni config update after the event monitor has been stopped... wondering if we should be closing the conf monitor first in this Close() func. last in first out :)

Reply:
Sure. Updated. I call the `c.cniNetConfMonitor.stop()` first and log error instead of return error. It is not blocked issue during shutdown period.